support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -142,16 +139,20 @@ class CreateSubscriptionController(
     }
   }
 
-  private def buildUser(user: IdUser, request: CreateSupportWorkersRequest, isTestUser: Boolean) = {
+  private def buildSupportWorkersUser(identityIdAndEmail: IdentityIdAndEmail, request: CreateSupportWorkersRequest, isTestUser: Boolean) = {
     User(
-      id = user.id,
-      primaryEmailAddress = user.primaryEmailAddress,
+      id = identityIdAndEmail.id,
+      primaryEmailAddress = identityIdAndEmail.primaryEmailAddress,
       title = request.title,
       firstName = request.firstName,
       lastName = request.lastName,
       billingAddress = request.billingAddress,
       deliveryAddress = request.deliveryAddress,
-      telephoneNumber = request.telephoneNumber,
+      telephoneNumber = for {
+          phoneNo <- request.telephoneNumber
+          updatedNo <- NormalisedTelephoneNumber.formatFromStringAndCountry(phoneNo, request.billingAddress.country)
+            .tap(_.left.foreach(SafeLogger.warn)).toOption
```

Comment:
we now get an Either from the normaliser, and can emit the warning at the point we discard the error, which makes more sense to me.